src/resources/postcss/components/_small-cta.pcss

```diff
@@ -0,0 +1,31 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Small CTA (Tooltips - ET, ET+, RSVP)
+ *
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.tribe-events-c-small-cta-link {
+		@mixin desktop-body-3-bold;
+
+		border-bottom: 2px solid var(--color-border-default);
+		cursor: pointer;
+		display: inline;
+		padding-bottom: 2px;
```

Comment:
this might fix the `padding-bottom` if we move away from `inline`. The `line-height` from the `desktop-body` mixin should provide the space required for that bottom border.